package.json

```diff
@@ -1,24 +1,28 @@
 {
   "name": "text-filter",
-  "version": "1.2.1",
+  "version": "2.2.5",
   "scripts": {
     "pbiviz": "pbiviz",
     "start": "pbiviz start",
     "package": "pbiviz package",
-    "lint": "tslint -c tslint.json -p tsconfig.json"
+    "lint": "npm run eslint",
+    "eslint": "npx eslint . --ext .js,.jsx,.ts,.tsx"
   },
   "devDependencies": {
-    "@babel/runtime": "7.6.0",
-    "@babel/runtime-corejs2": "7.6.0",
-    "@types/d3": "^5.16.4",
-    "core-js": "3.2.1",
-    "d3": "^5.16.0",
+    "@types/d3-selection": "^3.0.5",
+    "@typescript-eslint/eslint-plugin": "^5.59.1",
+    "@typescript-eslint/parser": "^5.59.1",
+    "eslint": "^8.39.0",
+    "eslint-plugin-powerbi-visuals": "^0.8.1",
     "powerbi-models": "^1.7.0",
-    "powerbi-visuals-api": "^3.4.0",
-    "powerbi-visuals-utils-dataviewutils": "2.2.1",
+    "powerbi-visuals-api": "~5.3.0",
+    "powerbi-visuals-tools": "^4.3.2",
+    "powerbi-visuals-utils-dataviewutils": "^6.0.1",
```

Comment:
move it to dependecies:
powerbi-visuals-api
powerbi-visuals-utils-dataviewutils
powerbi-visuals-utils-formattingmodel